css/styles.css

```diff
@@ -0,0 +1,57 @@
+body {
+    background: #FFF;
+}
+
+.page-header {
+    background: #FFF;
+    box-shadow: 0px 4px 4px 0px rgba(0, 0, 0, 0.25);
+}
+
+.header-icon {
+    width: 151px;
+    height: 101px;
+    flex-shrink: 0;
+}
+
+.container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+
+}
+
+.navigation {
+    display: flex;
+    align-items: center;
```

Comment:
see in dev tool this tag - the icon doesn't fit completely in it

Reply:
<img width="578" alt="image" src="https://github.com/artem-palytsia/github-hotel_project/assets/115567693/a5a4adb7-2c0d-4725-bdd7-c6851595ac93">